lib/ace/mode/latex_parser.js

```diff
@@ -33,26 +52,58 @@ var LatexParser = exports.LatexParser = function(bgTokenizer) {
                 }
 
                 switch(token.type) {
-                    case "text": case "comment":
-                        var words = this.splitTextToWords(token.value, row, column);
-                        for (var i = 0, wordsLength = words.length; i < wordsLength; ++i) {
-                            if (!dictionary.check(words[i].value)) {
-                                this.myErrors.push({
-                                    row: words[i].row,
-                                    column: words[i].column,
-                                    text: "grammar error",
-                                    type: "error",
-                                    raw: words[i].value
-                                });
-                            }
-                        }
+                    case "comment":
+                        doGetComments && parseRowForTextComments.call(this, token.value, row);
```

Comment:
Yes, a single row of text. Empty row is just a delimiter of paragraphs. Comment ends when we met another comment or when comments block ends.